docs/src/app/components/pages/components/GridList/Page.jsx

```diff
@@ -12,7 +12,7 @@ import gridListExampleComplexCode from '!raw!./ExampleComplex';
 import GridListExampleComplex from './ExampleComplex';
 
 const descriptions = {
-  simple: 'A simple example of a scrollable `GridList`.',
+  simple: 'A simple example of a scrollable `GridList` and GridList `Subheader`.',
```

Comment:
Should this be like: A simple example of a scrollable `GridList` containing a `Subheader`.